### ISSUE TITLE
event emitter leak

### ISSUE THREAD

**Issue by mafintosh** [moose-team/friends#70]
starting friends produces the following warning for me currently, 

```
[5949:0429/234914:INFO:CONSOLE(232)] "(node) warning: possible EventEmitter memory leak detected. %d %s listeners added. Use emitter.setMaxListeners() to increase limit.", source: events.js (232)
```

**Comment by shama** [comment]
Sounds like an event listener might be being registered in the render loop.

**Comment by shama** [comment]
Looks like this is the event that gets added a bunch: https://github.com/mafintosh/hyperlog/blob/master/index.js#L193

**Comment by mafintosh** [comment]
Fixed in hyperlog 3.4.1
